src/Resources/skeleton/verifyEmail/VerifyEmail.tpl.php

```diff
@@ -0,0 +1,54 @@
+<?= "<?php\n" ?>
+
+namespace <?= $namespace; ?>;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bridge\Twig\Mime\TemplatedEmail;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+use SymfonyCasts\Bundle\VerifyEmail\Exception\VerifyEmailExceptionInterface;
+use SymfonyCasts\Bundle\VerifyEmail\VerifyEmailHelperInterface;
+
+class <?= $class_name; ?><?= "\n" ?>
+{
+    private $helper;
+    private $mailer;
+    private $manager;
+
+    public function __construct(VerifyEmailHelperInterface $helper, MailerInterface $mailer, EntityManagerInterface $manager)
+    {
+        $this->helper = $helper;
+        $this->mailer = $mailer;
+        $this->manager = $manager;
+    }
+
+    public function sendEmailConfirmation(string $validationRouteName, UserInterface $user, TemplatedEmail $email): void
+    {
+        $signatureComponents = $this->helper->generateSignature(
+            $validationRouteName,
+            $user-><?= $id_getter ?>(),
+            $user-><?= $email_getter ?>()
+        );
+
+        $context = $email->getContext();
+        $context['signedUriComponents'] = $signatureComponents;
```

Comment:
WDYT about passing in only he `signedUrl` string and `expiresAt` instead of `signedUriComponents` - that "components" thing feels a bit abstract.

Reply:
done